src/main/resources/admin/tools/content-type-manager/content-type-manager.html

```diff
@@ -7,6 +7,19 @@
 
     <title>Content Type Manager</title>
 
+    <link rel="stylesheet" type="text/css" data-th-href="|${assetsUrl}/admin/common/styles/lib.css|">
+    <script data-th-inline="javascript">
+      var CONFIG = {
+          launcherUrl: /*[[${launcherUrl}]]*/ '', // Required for launcher
+          launcher: {
+            theme: 'light'
+          }
+      };
+    </script>
+    <script data-th-src="|${assetsUrl}/admin/common/js/lib.js|"></script>
+    <script async data-th-src="${launcherPath}" type="text/javascript"></script>
```

Comment:
Seems that this is responsible for this error:
![image](https://user-images.githubusercontent.com/67838246/144294817-1e2d9a60-3035-4edb-a6bc-f0f942c030a5.png)

Reply:
This should be fixed in the latest version by Alan

Reply:
This is a tricky one. Vue app object added to the global `window` object (`window.app`) breaks some logic which we have inside `lib-admin-ui` (used by the Launcher button which Persijn added). I added a check to `lib-admin-ui` which solves the issue but we need to release that fix (in XP 7.8) and then change dependency in `app-cty-manager` to require XP 7.8 or later. 

Two other alternatives (which won't force users to upgrade to XP 7.8) are:
* remove the XP launcher button
* fix the way the app is created so that it's not added to the global `window` object (this one we'll have to do at some point anyway, so maybe this is a way to go).